JavaAssignments/src/amarja/CalculatorProg.java

```diff
@@ -0,0 +1,31 @@
+package amarja;
+//Program to find addition,subtraction,multiplication and division of two numbers
+public class CalculatorProg {
+		public static void main(String[] args) {
+		CalculatorProg a = new CalculatorProg();
+		System.out.println("Number 1= "+ Integer.parseInt(args[0]));
+		System.out.println("Number 2= "+ Integer.parseInt(args[1]));
+		a.addition(Integer.parseInt(args[0]),Integer.parseInt(args[1]));
+		a.subtraction(Integer.parseInt(args[0]),Integer.parseInt(args[1]));
+		a.multiplication(Integer.parseInt(args[0]),Integer.parseInt(args[1]));
+		a.division(Float.parseFloat(args[0]),Float.parseFloat(args[1]));
+	}
+		void addition (int num1,int num2)
```

Comment:
it is always better to return result, caller may expect some result while calling it.

Reply:
ok will follow always

Reply:
Amarja, change it here also. change return type void to int or float.